packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/tests/index.test.js

```diff
@@ -110,6 +108,6 @@ describe('BulkActionsBar', () => {
       );
     });
 
-    expect(onConfirmUnpublishAll).toHaveBeenCalledWith([]);
+    expect(screen.getByText(/Are you sure you want to unpublish these entries?/)).toBeVisible();
```

Comment:
Are you testing anywhere that your callbacks are being fired with expected values?